cibuildwheel/windows.py

```diff
@@ -172,7 +172,8 @@ def pep_518_cp35_workaround(package_dir: Path, env: Dict[str, str]) -> None:
             else []
         )
         if requirements:
-            shell(['pip', 'install'] + requirements, env=env)
+            escaped_requirements = [f'"{s}"' for s in requirements]
```

Comment:
Thinking out loud: is this something we should do here, or more broadly inside of `shell`?

Reply:
We don't have any protection for quotes inside the string, so I was a little hesitant to add it higher up. If someone wrote `"requests==2.22.0; python_version<\"3.6\"",`, in fact, this might not work.

Reply:
Good point. I suppose we also control most of the other things to not contain spaces?
Though .. what about spaces in `CIBW_TEST_EXTRAS` or so?

Reply:
That can be changed by the cibw user, if something odd happens. You can't change `pyproject.toml` (at least you shouldn't in CI).